nodewatcher/modules/analysis/channel_allocation/channel_lookup.py

```diff
@@ -0,0 +1,41 @@
+from nodewatcher.core.monitor import models
+
+
+def get_channel(node_bssid):
+    """
+    Gets the current channel of the BSSID
+
+    :param node_bssid:
+    :return:
+    """
+
+    return models.WifiInterfaceMonitor.objects.filter(bssid=node_bssid)[0].channel
+
+
+def is_2ghz_bssid(node_bssid):
+    """
+    Is this BSSID associated to 2.4ghz or 5ghz?
+
+    :param node_bssid:
+    :return: Boolean whether the BSSID belongs to the 2.4ghz spectrum or not.
+    """
+
+    return models.WifiInterfaceMonitor.objects.filter(bssid=node_bssid)[0].channel <= 11
+
+
+def get_available_channels(node_bssid):
+    """
+    Returns an array of available channels for that node.
+
+    :param node_bssid: BSSID of a node
+    :return: Array of available channels.
+    """
+    highest_2ghz_channel = 11
+    ch_2ghz = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
```

Comment:
The issue is that some of these channels are not available to the nodes in the US (specifically, 12 and 13 in the 2.4GHz band). Mitar and I think we should add a node-specific subset of available channels. But in the meantime, should we leave the hard-coded array or do something else?